packages/nodes-from-anchor/src/v01/AccountNode.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    AccountNode,
+    accountNode,
+    assertIsNode,
+    bytesTypeNode,
+    camelCase,
+    fieldDiscriminatorNode,
+    structFieldTypeNode,
+    StructTypeNode,
+    structTypeNode,
+} from '@kinobi-so/nodes';
+
+import { getAnchorDiscriminatorV01 } from '../discriminators';
+import { IdlV01Account, IdlV01TypeDef, IdlV01TypeDefTyStruct } from './idl';
+import { structTypeNodeFromAnchorV01 } from './typeNodes';
+
+export function accountNodeFromAnchorV01(idl: IdlV01Account, types: IdlV01TypeDef[]): AccountNode<StructTypeNode> {
+    const definition = types.find(type => type.name === idl.name) as IdlV01TypeDef;
+
+    const idlName = definition.name;
+    const name = camelCase(idlName);
+
+    // TODO: support enum and generic types.
+    let data = structTypeNodeFromAnchorV01(definition.type as IdlV01TypeDefTyStruct);
+    assertIsNode(data, 'structTypeNode');
```

Comment:
Yeah currently I'm not planning on supporting enum type for accounts because we'd loose a lot of the connections within the Kinobi tree if we did that but I'll simmer on it.

Reply:
There is a enum type node currently should that be dropped? I need to look over the enum structure some more.